src/commands.cpp

```diff
@@ -4503,12 +4498,14 @@ void CommandChatCreate::procresult()
                         TextChat *chat = client->chats[chatid];
                         chat->id = chatid;
                         chat->priv = PRIV_MODERATOR;
-                        chat->url = url;
                         chat->shard = shard;
                         delete chat->userpriv;  // discard any existing `userpriv`
                         chat->userpriv = this->chatPeers;
                         chat->group = group;
 
+                        chat->setTag(tag ? tag : -1);
```

Comment:
I think that this check and other similar ones in this pull request could be removed because the tag `0` shouldn't be used (except for commands generated internally by the SDK without any interest for client apps and libraries), but I understand that they ensure that `isOwnChange()` never returns `0` for own changes, even if a wrong tag is set.